openshift/app-oidc.dc.yaml

```diff
@@ -0,0 +1,316 @@
+---
+apiVersion: v1
+kind: Template
+labels:
+  app.kubernetes.io/component: app
+  app.kubernetes.io/instance: "${APP_NAME}-${JOB_NAME}"
+  app.kubernetes.io/managed-by: jenkins
+  app.kubernetes.io/name: nodejs
+  app.kubernetes.io/part-of: "${APP_NAME}-${JOB_NAME}"
+  app: "${APP_NAME}-${JOB_NAME}"
+  template: "${REPO_NAME}-app-dc-template"
```

Comment:
do you want template and name to be the same as the current?  querying by template or name will bring back so and oidc objects?

Reply:
Yes - normally the template label is to signify what deployment config template is managing this. We're hand-bombing this at the moment and neither our pipeline nor our cleanup instructions actually use `template` or `app.kubernetes.io/name` label selectors to identify the resources, We still select by just the `app` label in nearly all cases.

As an aside, label `app.kubernetes.io/name: nodejs` will be used in OCP4 for making the topology view have the correct nodejs icon representing the deployment config.

Reply:
ok, just don't want to accidentally remove the wrong resources.